airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+public class DbMigrationApiController {
```

Comment:
what is this for?

Reply:
that's a mistake, It was in my untracked git change and sneak in the PR. I'll remove it once I have a green build.

Reply:
👍 this looks good to me otherwise so i'll approve